helpers/components/addOrderedIds.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const _ = require('lodash');
+
+/**
+ * Add ordered ids to components within a componentlist
+ * @param {Array} content   list of components
```

Comment:
When I changed `{Array}` to `{Object[]}`, I got this error when trying to push:
```
throw new Error(`Unknown param type "${obj.type}"!`);
```
I guess it failed to generate the readme with that `Object[]` param..?